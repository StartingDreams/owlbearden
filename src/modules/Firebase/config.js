const config = {
  apiKey: 'AIzaSyDg4Z-8ROBMuRtzU5ThnU1GMd2IWgsttEU',
  authDomain: 'owlbearsden.firebaseapp.com',
  databaseURL: 'https://owlbearsden.firebaseio.com',
  projectId: 'owlbearsden',
  storageBucket: 'owlbearsden.appspot.com',
  messagingSenderId: '149275717879',
};

export default config;
