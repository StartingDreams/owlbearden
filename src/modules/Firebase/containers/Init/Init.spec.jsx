import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import renderer from 'react-test-renderer';
import { Init } from './Init';
import config from '../../config';

jest.mock('firebase', () => {
  const initializeApp = jest.fn();
  return { initializeApp };
});

const firebase = require('firebase');

describe('<Init />', () => {
  it('spell should match the snapshot', () => {
    const wrapper = shallow(<Init />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('login should be dispatched', () => {
    renderer.create(<Init />);
    expect(firebase.initializeApp).toHaveBeenCalledWith(config);
  });
});
