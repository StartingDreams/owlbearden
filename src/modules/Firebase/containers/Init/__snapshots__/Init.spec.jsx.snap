// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Init /> spell should match the snapshot 1`] = `null`;
